randovania/data/json_data/prime3.txt

```diff
@@ -2067,59 +2427,86 @@ Hall of Remembrance
 Asset id: 6915612318130423578
```

Comment:
For rooms above where I can't comment:

- Warp Site Alpha
  - Add node called "Event - Warp Site Alpha Wall". Connect to Dock to Main Lift as "Trivial"
  - Dock to Main Lift
    - Change to Plasma Door
    - Connect to "Event - Warp Site Alpha Wall" with requirements Boost Ball and (Plasma Beam or Nova Beam)
  - Add node called "Teleport to Bryyo Ice - Warp Site Bravo".
    - Connect to Dock to Main Lift with requirement "After Warp Site Alpha Wall" (See "Event 21" in prime3.json for more details)

Reply:
You still haven't added the changes to warp site bravo as mentioned